cmd/kubeops/internal/handler/handler_test.go

```diff
@@ -182,7 +182,6 @@ func TestActions(t *testing.T) {
 			Description: "Get a simple release",
 			ExistingReleases: []*release.Release{
 				createRelease("foo", "foobar", "default", 1, release.StatusDeployed),
-				createRelease("oof", "oofbar", "dev", 1, release.StatusDeployed),
```

Comment:
do you need to delete these or it was just for debugging purposes?

Reply:
It was useless now that the memory driver is namespace aware, since it is never seen by the test (unless you explicitly changed the namespace to "dev").